core/api/src/graphql/public/queries.ts

```diff
@@ -27,6 +28,7 @@ export const queryFields = {
     realtimePrice: RealtimePriceQuery,
     btcPriceList: BtcPriceListQuery,
     lnInvoicePaymentStatus: LnInvoicePaymentStatusQuery,
```

Comment:
I wonder if we should do something like this just so it's clearer internally?

```suggestion
    lnInvoicePaymentStatus: LnInvoicePaymentStatusByPaymentRequestQuery,
```

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/3899/commits/0f67fbcf9456930df5d2d5fc09ebdb2dedd3f9f7